### PR TITLE
Add checks for AirBrakeMaxMainResPipePressure.

### PR DESCRIPTION
**Pull request by peternewell**
https://blueprints.launchpad.net/or/+spec/brake-enhancements

Corrects a "further" issue identified -

Bug report: https://www.elvastower.com/forums/index.php?/topic/35558-compressor-doesnt-shut-off/

### CODE REVIEW

**Review by Sharpe49 — CHANGES_REQUESTED** [review]

**Review by Sharpe49 — APPROVED** [review]
Code is correct.
Waiting for @cjakeman approval to merge it.

**Comment by peternewell** [comment]
Thanks for that.

**Pull request merged**
